manifests/crane.pp

```diff
@@ -20,17 +22,23 @@
 #
 # $ssl_protocol::               SSLProtocol configuration to use
 class pulp::crane (
+  Stdlib::Absolutepath $key,
+  Stdlib::Absolutepath $cert,
+  Stdlib::Absolutepath $ca_cert,
```

Comment:
The parameters aren't `Optional`, so I don't see how it makes sense to default them to `undef` (via params.pp)